LocalGoodwillOnline/src/java/com/mycompany/controllers/SearchedItemController.java

```diff
@@ -1,244 +0,0 @@
-package com.mycompany.controllers;
```

Comment:
deleting searchedItemController? Maybe you were not uptodate with master?

Reply:
weird i did this after i pushed search and made the pull request.